src/merge.py

```
@@ -544,43 +581,17 @@ def main():
-                    matched.action = EventAction.none
-                    icloud_event.action = EventAction.none
-                else:
-                    icloud_event.action = EventAction.delete
-                merge_events.append(icloud_event)
-
-            events_to_add = [event for event in source_calendar_events if event.action is None]
-            event_addition = len(events_to_add) > 0
-        else:
-            event_addition = True
-            events_to_add = source_calendar_events
+        print_step(TAG_CALENDAR_MERGE, "reconciling events...", False)
```

Ruff (0.15.7)

[warning] Boolean positional value in function call

(FBT003)